core/variant_op.cpp

```diff
@@ -1057,7 +1057,7 @@ void Variant::evaluate(const Operator &p_op, const Variant &p_a,
 					_RETURN("Division By Zero");
 				}
 #endif
-				_RETURN(p_a._data._int % p_b._data._int);
+				_RETURN(((p_a._data._int % p_b._data._int) + p_b._data._int) % p_b._data._int);
```

Comment:
An explanatory comment is direly needed here.

Reply:
Sure :)

Reply:
Done :)